pages/1.13/security/oss/authentication/index.md

```diff
@@ -0,0 +1,43 @@
+---
+layout: layout.pug
+navigationTitle:  Authentication
+title: Authentication
+excerpt: Authenticating users against DC/OS
+menuWeight: 30
+
+---
+
+<!-- The source repository for this topic is https://github.com/dcos/dcos-docs-site -->
+
+# Authentication in DC/OS
+
+In DC/OS user authentication is required by default.
```

Comment:
Combine line #14,#15,#16 into one line. 
In DC/OS, user authentication is required by default. Every user who wants to perform an operation on a DC/OS cluster (other than login) must first be authenticated.